File: documentation-examples-kotlin/src/kotlin/dev/typr/foundations/docs/oracle/RowIdTypes.kt
package dev.typr.foundations.docs.oracle

import dev.typr.kotlinfoundations.*
import dev.typr.kotlinfoundations.data.*

@Suppress("unused")
class RowIdTypes {
    //start
    val rowidType: OracleType<String> = OracleTypes.rowId
    val urowidType: OracleType<String> = OracleTypes.uRowId
    val urowid1000: OracleType<String> = OracleTypes.uRowId(1000)
    //stop
}

File: documentation-examples-scala/src/scala/dev/typr/foundationssc/docs/oracle/RowIdTypes.scala
package dev.typr.foundationssc.docs.oracle
import dev.typr.foundationssc.*
import dev.typr.foundationssc.data.*



@SuppressWarnings(Array("unused"))
object RowIdTypes:
  //start
  val rowidType: OracleType[String] = OracleTypes.rowId
  val urowidType: OracleType[String] = OracleTypes.uRowId
  val urowid1000: OracleType[String] = OracleTypes.uRowId(1000)
  //stop
